examples/platform/silabs/display/lcd.cpp

```diff
@@ -126,7 +128,30 @@ int SilabsLCD::DrawPixel(void * pContext, int32_t x, int32_t y)
 
 int SilabsLCD::Update(void)
 {
-    return updateDisplay();
+    int status = 0;
+#ifdef SL_ENABLE_ICD_LCD
```

Comment:
I don't see where this is defined But I'd also suggest a more specific name to what it enables.
SL_MATTER_TIMED_SCREEN_DISPLAY